benefits/static/css/styles.css

```diff
@@ -245,46 +253,67 @@ footer .footer-links li a:visited {
 }
 
 /* Buttons */
+/* Primary Button: Use all three classes: btn btn-lg btn-primary */
+/* Set button width in parent with Bootstrap column */
+/* Height: 60px on Desktop; 72 on mobile*/
 
-.btn-lg {
-  border-width: 2px;
+:root {
+  --primary-button-padding: 19px 0;
 }
 
-.btn-link {
-  color: var(--primary-color);
+@media (min-width: 992px) {
+  :root {
+    --primary-button-padding: 13px 0;
+  }
 }
 
-.btn-primary {
+.btn-lg.btn-primary.btn-lg {
+  border-width: 2px;
   text-transform: capitalize;
-  letter-spacing: 0.05em;
+  letter-spacing: 0.02em;
+  font-weight: var(--medium-font-weight);
+  font-size: calc(20rem / 16);
+  width: 100%;
+  padding: var(--primary-button-padding);
 }
 
+/* A button that actually is a text link */
+/* TODO: Check if this is actually really used anymore */
```

Comment:
It looks like we do here: https://github.com/cal-itp/benefits/blob/dev/benefits/eligibility/views.py#L92

But I'm not sure if that is needed...

Reply:
Yeah I think this will eventually get deleted. But seemed outside of scope of this PR. I think whoever touches `eligibility#start` next should delete it.